recipes/stgit/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "stgit" %}
+{% set version = "2.4.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/stacked-git/stgit/archive/v{{ version }}.tar.gz
+  sha256: 087ec6b4aec92522687ede9fb249c98ac49e183f8bdd58c8015deaf69919d7e3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - cargo-bundle-licenses
+    - make
+    - pkg-config
+  host:
+    - curl
+    - git
+    - zlib
```

You should not need to have a `libcurl` run dependency explicitly, but it should rather come in via `run_exports`
```suggestion
    - zlib
    - libcurl
```

---

Please see the last commit for this change.